zest/releaser/tests/pypi.txt

```diff
@@ -43,22 +43,6 @@ There are two styles of ``.pypirc`` files.  The old one just for pypi:
     >>> pypiconfig.distutils_servers()
     ['pypi']
 
-With ``get_server_config`` we get the settings that we pass to ``twine``:
-
-    >>> from pprint import pprint
```

Comment:
This breaks some tests later on.

Reply:
That's curious. Tests pass locally.  Anyway I see you fixed this in e7742a67f1d6